apis/kueue/v1alpha1/localqueue_types.go

```diff
@@ -42,24 +42,24 @@ type QueueStatus struct {
 //+kubebuilder:printcolumn:name="ClusterQueue",JSONPath=".spec.clusterQueue",type=string,description="Backing ClusterQueue"
 //+kubebuilder:printcolumn:name="Pending Workloads",JSONPath=".status.pendingWorkloads",type=integer,description="Number of pending workloads"
```

Comment:
Worth adding a short name:

```
//+kubebuilder:resource:shortName={queue,queues}
```

Reply:
should we use that for clusterQueue :)

Reply:
Uhm... I don't think so. An end user should be able to do `kubectl get queues` to get the names of the queues they can use in `Workload.spec.queueName`